brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Trainer.scala

```diff
@@ -1,6 +1,6 @@
-package com.stripe.brushfire.scalding
+package com.stripe.brushfire
+package scalding
```

Comment:
It doesn't seem to be popular, but I don't think most people really think about it (especially since the package statement is usually generated). I like it because it kind of makes the relationship a bit more clear, but I don't think there is a practical difference between this vs just 1 package statement + a wildcard import.